File: PHP/card.php
<?php
require_once('payment.php');
class Card extends Payment{
    public $cardNumber;
    public $secureCode;
    public $mark;

    public function __construct($id, $value, $cardNumber, $secureCode, $mark ){
        parent::__construct($id, $value);
        $this->cardNumber = $cardNumber;
        $this->secureCode = $secureCode;
        $this->mark = $mark;
    }
}

File: java/Card.java
class Card extends Payment {

    Integer value;
    String cardNumber;
    String secureCode;
    String mark; 

    public Card (Integer value, String cardNumber, String secureCode, String mark, Integer id){
        super(id);
        this.value = value;
        this.cardNumber = cardNumber;
        this.secureCode = secureCode;
        this.mark = mark;
    }
    
}

